package.json

```diff
@@ -109,6 +109,16 @@
       "default": true,
       "order": 6
     },
+    "loggingLevel": {
+      "type": "string",
+      "enum": [
+        "info",
+        "warning",
+        "error"
```

Comment:
Fixed, and I added a description which is not so brief. I can remove the "such as ..." if you want to make it more terse.